jaeger-operator/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM golang:1.13-alpine as builder
+
+ENV JAEGER_OPERATOR_VERSION 1.17.1
+
+WORKDIR /go/src/github.com/jaegertracing
+
+RUN apk update && apk upgrade && \
+    apk add --no-cache bash git openssl make
+
+RUN git clone --branch v${JAEGER_OPERATOR_VERSION} --single-branch --depth 1 https://github.com/jaegertracing/jaeger-operator.git
```

Comment:
Isn't it easier from maintenance perspective to just copy the binaries from original image instead of building them here?